webapp/src/main/webapp/vue-apps/CallButton/components/JitsiMeetButton.vue

```diff
@@ -32,35 +37,30 @@ export default {
       settings: this.callSettings,
       log: null,
       callWindow: null
-      // callState: null
     };
   },
   computed: {
     callState: function() {
       return this.callSettings.callState;
     },
+    parentElement() {
+      if (this.$refs.jitsi) {
+        return this.$refs.jitsi.$el.parentElement.parentElement.parentElement.parentElement.parentElement;
```

Comment:
Isn't it possible to pass this information using a component props ? that will avoid navigating through the DOM

Reply:
We changed getting of this information (actually getting parent classes) to this.callSettings.context.parentClasses, edited and forcepushed the commit

Reply:
Looks good, thanks !

Reply:
Thank you for the review